src/message/messageActionSheet.js

```diff
@@ -274,11 +283,28 @@ export const showActionSheet = (
       }
     })();
   };
+  let title: string;
+  if (Array.isArray(params.message.display_recipient)) {
+    // This means the message is a private message.
+    title = `${_('Private message between')} `;
+    params.message.display_recipient.forEach(
+      (recepient: PmRecipientUser, idx: number, recepients: PmRecipientUser[]) => {
+        title += `${recepient.full_name}`;
+        if (idx < recepients.length - 2) {
+          title += ', ';
+        } else if (idx === recepients.length - 2) {
+          title += ' & ';
+        }
+      },
+    );
```

Comment:
There are several issues here.

* We typically exclude the current user from the names of a group PM.
* `` `${someStringValue}` `` is redundant; it's equivalent to a bare `someStringValue`.
* The variable name `recepient` is misspelled (correctly: `recipient`).
* This is entirely localization-incompatible. In Japanese, for example, this would need to be something like `John Doeとのプライベートメッセー` and/or `John Doe、Welcome Bot、donut bot間のプライベートメッセー`: not only is different punctuation used, but the fundamental word order is different. String concatenation is, in general, unacceptable for localized text.

  At present, we actually give up on localization for this and use bare `names.join(', ')`, see e.g. [here](https://github.com/zulip/zulip-mobile/blob/28bd508eaedbbed35ca29486f5d363b222ff4cc3/src/pm-conversations/GroupPmConversationItem.js#L44). This is... not great,  but it mostly sidesteps the issue. Perhaps in the future we'll move to `Intl.ListFormat`.

  Also, Zulip prefers use of the Oxford comma (at least in the en-US locale).